File: To_DO_List_Python/tests/test_case.py
import unittest
from utilities.utility import Utility 
from pages.home_page import Home_Page 

class TestCase(unittest.TestCase):
    def setUp(self):
        self.chromeDriver = Utility.open_browser()

    def test_AddValidNewTask(self):
        page = Home_Page(self.chromeDriver)
        page.AddNewTask("Drink more water").MarkAsCompleted().DeleteCompletedTask()

    def tearDown(self):
        Utility.QuitTest(self.chromeDriver)

if __name__ == "__main__":
    unittest.main()

File: To_DO_List_Java/src/test/java/TestCase.java
import Pages.Home_Page;
import Utilities.Utility;
import org.openqa.selenium.WebDriver;
import org.testng.annotations.AfterMethod;
import org.testng.annotations.BeforeMethod;
import org.testng.annotations.Test;

public class TestCase {
    private WebDriver chromeDriver;

    @BeforeMethod
    public void SetupDriver(){
        chromeDriver = Utility.OpenBrowser(chromeDriver);
}

    @Test
    public void AddValidNewTask (){
        Home_Page page = new Home_Page(chromeDriver);
        page.AddNewTask("Drink more water").MarkAsCompleted().DeleteCompletedTask();
    }



    @AfterMethod
    public void Quit(){
    Utility.QuitTest(chromeDriver);
    }
}
